meetings/2023-05-10.md

```diff
@@ -0,0 +1,124 @@
+* eP: Yes. linked with "describedby"
+* SC: If you don't have human readable label, everything after the last slash is used
+* SC: One convention used when no RDF label is available, then use string after # or last /.
+* JZ: With the way URIs are built, can't do anything - not reliably reveal users' name. For practical reasons need to do something. In solidOs there is a method called label, and it tries to come up with a human-readable label. It is a bit more complicated than # / string but same idea.
+* eP: Shapes trees provide a way which predicate to use for the human.
+* eP: It is generic / not domain-specific. If you have FOAF, you'd specify in a shape tree, that foaf:name would be the one describing the label. The subject would be the identity of the resource and a particular predicate would be label.
+* SC: In dokeli before it gets to use string from URI, it tries some common properties and are commonly reused, for `foaf:` it makes sense to use for `foaf:name`, one can also try `rdfs:label`.
+* eP: in ShapeTrees we have advantage of conforming to a specific shape so a predicate can be used from the ones that are required by the shape.
+* SC: It depends if the model is more strict/controlled or is more free mix of vocabs. In HTML documents people may use schema.org or DC terms or RDFS. 
+* eP: on mobile, they tend to show just the domain
+* SC: or doesn't fit the screen even
+* eP: IIRC, Safari.
+
+
+### Change JSON-LD context URI in Solid Notifications Protocol
+
+* SC: https://github.com/solid/notifications/pull/174
+* SC: `https://www.w3.org/ns/solid/notifications-context/v1` will be the first published URL.
+* SC: No need for .jsonld right? Only available in JSON-LD any way.
+* eP: Yea, current practise is to not include it.
+* SC: If we publish spec and publish JSON-LD context, we are okay.
+* eP: Context can be published ASAP.
+* SC: Spec needs to refer to it before publishing. ED should be sufficient for now. We can nudge W3C Team to follow-up right after merging of 174 to publish the JSON-LD Context.
+
+
+### Solid Protocol Release 0.11.0
+URL: https://github.com/solid/specification/milestone/7
+
+* SC: Have started to mark items towards this milestone.
+* SC: Suggestions and commitments welcome.
+* SC: There are other milestones like FPWD, e.g., https://github.com/solid/specification/milestone/1 , which  I created a long time ago. These are good candidates towards 0.11.0. See also https://github.com/solid/specification/milestone/2 and https://github.com/solid/specification/milestone/3 ..
+* SC: Apparently also moving items from https://github.com/solid/specification/milestone/6 to https://github.com/solid/specification/milestone/7 .
+* SC: When I created these milestones, the idea was to take up the issues based on importance of publishing a REC in the WG. Those maturity levels don't have standing in the CG. If the WG happens it will need to do the same exercise since the maturity levels apply to the WG.
```

Comment:
```suggestion
* SC: When I created these milestones, the idea was to take up the issues based on importance of publishing a REC in the WG. Those maturity levels don't have standing in the CG. If the WG happens, it will need to do the same exercise, since the maturity levels apply to the WG.
```